cmd/kubelet/app/server.go

```diff
@@ -502,6 +502,20 @@ func InitializeTLS(kc *componentconfig.KubeletConfiguration) (*server.TLSOptions
 		CertFile: kc.TLSCertFile,
 		KeyFile:  kc.TLSPrivateKeyFile,
 	}
+
+	if ks.TLSCertAuth {
```

Comment:
Rather than setting these here, we should use the specified CA file to build an x509 authenticator and set the Authenticator interface in the kubelet serving options.